project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.3
+sbt.version=1.2.3
```

Comment:
Any idea why this file is showing a diff?

Reply:
File I originally copied had weird newline diff that I updated
https://superuser.com/questions/1217622/how-can-i-remove-the-m-from-my-file-in-sublime-text-3

affects some other files too e.g. HomeController